features/step_definitions/git-flow.js

```diff
@@ -80,6 +81,21 @@ defineSupportCode(({ Before, Given, When, Then }) => {
             });
     });
 
+    Given(/^a pipeline with all stopped builds$/, {
+        timeout: TIMEOUT
+    }, function step() {
```

Comment:
Do we need a return with Promise Object for this `Given` statement ?

Reply:
That's true. I fixed it.